src/include/miopen/solver/implicitgemm_ck_util.hpp

```diff
@@ -606,6 +643,12 @@ ConvSolution InitInvokerFactoryNCHW(const ExecutionContext& ctx,
                 std::swap(conv_tensors.xDesc, conv_tensors.yDesc);
             }
 
+            HipEventPtr start = nullptr;
```

Comment:
Quote from https://github.com/ROCm/MIOpen/pull/2770#discussion_r1506089187:

> When the ck-profiling is enabled, the ck wrw kernel introduces the errors that cause the precision issue.

- Please open an issue in CK repo and add a link to it to the description of this PR.
- Please create the `WORKAROUND_CK_ISSUE_XXX` with value 1 and wrap all the W/A related code in `#if WORKAROUND_CK_ISSUE_XXX ... #endif`. See #1685 for more info.

Reply:
> Quote from [#2770 (comment)](https://github.com/ROCm/MIOpen/pull/2770#discussion_r1506089187):
> 
> > When the ck-profiling is enabled, the ck wrw kernel introduces the errors that cause the precision issue.
> 
> * Please open an issue in CK repo and add a link to it to the description of this PR.
> * Please create the `WORKAROUND_CK_ISSUE_XXX` with value 1 and wrap all the W/A related code in `#if WORKAROUND_CK_ISSUE_XXX ... #endif`. See [Lifecycle of workarounds #1685](https://github.com/ROCm/MIOpen/discussions/1685) for more info.

Yes. I am about to do it. I have discussed with CK team about it. It seems that the issue I found is a none trivial task for them. That is why I am trying to find some workaround solution from hostside.